README.md

```diff
@@ -1 +1,78 @@
-# loofah-graph-api
\ No newline at end of file
+# LOOFAH GRAPH API
+
+## Overview
+
+A Graph QL API which returns a list of skills
+
+## Usage
+
+**Endpoint**: `localhost:8080/skills`
+
+**Method**: `GET`
```

Comment:
I've set this up as a GET request but, given the potential complexity of GET queries, I'm not sure that it shouldn't be a POST. I did actually make the change to GET quite late in the day having read [this section of the docs](https://graphql.org/learn/serving-over-http/) so I've retained the POST version of this code (incl integration tests) on [a separate branch](https://github.com/elishkaflint/loofah-graph-api/tree/LOOFAH-POST-ROUTE), in case it's useful for future reference.

Reply:
My first instinct is that it's neater to have the query in the body of the request than the query string, especially as it gets more complex. Also I presume we'd want to use POST for various mutations etc for security, so using POST for both might add consistency. What do you think?

Reply:
I agree - I was really in two minds but also mainly wanted to get my head around how each would work. I've reverted to using POST for all requests.